proto/BUILD.bazel

```diff
@@ -100,6 +91,21 @@ py_proto_library(
     deps = [":p4runtime_proto"],
 )
 
+go_proto_library(
+    name = "p4info_go_proto",
+    importpath = "github.com/p4lang/p4runtime/go/p4/config/v1",
+    protos = [
+        ":p4info_proto",
+        ":p4types_proto",
+    ],
+)
+
+go_proto_library(
+    name = "p4runtime_go_proto",
+    importpath = "github.com/p4lang/p4runtime/go/p4/v1",
+    protos = [":p4runtime_proto"],
```

Comment:
The build is currently failing. I believe we have two include some additional dependencies here:

```
compilepkg: missing strict dependencies:
	/home/runner/.cache/bazel/_bazel_runner/1fc64d2e702c0d08ecdb88fe41750bb3/sandbox/linux-sandbox/2/execroot/com_github_p4lang_p4runtime/bazel-out/k8-fastbuild/bin/p4runtime_go_proto_/github.com/p4lang/p4runtime/go/p4/v1/p4runtime.pb.go: import of "github.com/p4lang/p4runtime/go/p4/config/v1"
	/home/runner/.cache/bazel/_bazel_runner/1fc64d2e702c0d08ecdb88fe41750bb3/sandbox/linux-sandbox/2/execroot/com_github_p4lang_p4runtime/bazel-out/k8-fastbuild/bin/p4runtime_go_proto_/github.com/p4lang/p4runtime/go/p4/v1/p4runtime.pb.go: import of "google.golang.org/genproto/googleapis/rpc/status"
No dependencies were provided.
```